### DISCUSSION TITLE
Use cases for computed_field

### DISCUSSION THREAD

**Discussion by FidelusAleksander** [pydantic/pydantic#6509] [Pydantic V2]
I'm trying to figure out when (or if) should I use @computed_field decorators instead of just raw @property on BaseModels

This is one of my use cases, simplified.
```
class Model(BaseModel):
    delay_seconds: int

    @computed_field
    def high_delay(self) -> bool:
        return self.delay_seconds > 5

class Model(BaseModel):
    delay_seconds: int

    @property
    def high_delay(self) -> bool:
        return self.delay_seconds > 5
```

How are You guys using the computed_field decorator?

**Comment by adriangb** [comment]
If you use `@property` it will not be included in your json schema, when you serialize to json, etc.
